apps/nightscout/nightscout.star

```diff
@@ -41,7 +41,7 @@ GRAPH_TOP = 275
 CACHE_TTL_SECONDS = 1800  #30 mins
 
 PROVIDER_CACHE_TTL = 7200  #2 hours
-NS_PROVIDERS = "https://gist.githubusercontent.com/IsThisPaul/e976c41112d79383c51ed7315bb114ab/raw/170f967eba27587ffd289631c1a2ae9e31f1bee3/nightscout_providers.csv"
+NS_PROVIDERS = "https://raw.githubusercontent.com/IsThisPaul/TidBytCommunity/Nightscout-Provider-File/apps/nightscout/nightscout_providers.csv"
```

Comment:
Spoke with @IsThisPaul on Discord, we're going to give this a go for now. @IsThisPaul will validate that these are actual Nightscout instances and maintain the list. It does open us up to somewhat arbitrary domains, but I'd rather fix an issue on our side if we discover one then be a blocker for Nightscout users.